### PR TITLE
Term Entry for Python: NumPy function .amin(), closes issue #2094

### PR DESCRIPTION
**Pull request by teona-pavliukova**
### Description

A new term entry was created for [Python:NumPy](https://www.codecademy.com/resources/docs/numpy) **.amin()** function in a new file in `docs/content/numpy/concepts/built-in-functions/terms/min/amin.md`.  This entry closses issue #2094.

The entry have all the below requirements listed on the issue's page:
- Mention the equivalent ndarray.min() method.
- Have a syntax section illustrating the function with a ```pseudo code block.
- An example showing the function in use with a ```py code block.
- A runnable codebyte example in a  ```codebyte/python block.

An entry was created as part of a portfolio project for *Learn Git and GitHub* course.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/numpy/concepts/built-in-functions/terms/min/amin.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* `keepdims`: a boolean; if `True` will keep reduced axes in the result as dimensions with size one.
+* `inital`: The maximum value of an output element.
+* `where`: A boolean array that maps to `array` and selects which elements to compare for the minimum.
+
+
+Returns:
+
+* If axis is None, the result would be a scalar value of the minimum element of the array.
+* If axis is an integer, the result would be an array of dimension a.ndim -1.
+* If axis is a tuple, the result would be an array of dimension a.ndim - len(axis)
```
Comment (caupolicandiaz) [inline content/numpy/concepts/built-in-functions/terms/min/amin.md:44]:
```suggestion
* If axis is an integer, the result would be an array of dimension `a.ndim - 1`.
* If axis is a tuple, the result would be an array of dimension `a.ndim - len(axis)`.
```

File: content/numpy/concepts/built-in-functions/terms/min/amin.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Parameters of `.amin()` function:
+
+* `a`: The array of elements to choose the minimum from.
+* `axis`: An int or a tuple of ints specifying the axis/axes along which to choose the minimum. By default the flattened input is used (`axis=None`).
+* `out`: An [`ndarray`](https://www.codecademy.com/resources/docs/numpy/ndarray) to receive result. Must have the same shape as expected output.
+* `keepdims`: a boolean; if `True` will keep reduced axes in the result as dimensions with size one.
+* `inital`: The maximum value of an output element.
+* `where`: A boolean array that maps to `array` and selects which elements to compare for the minimum.
+
```
[the remark this answers is not in the document]
Comment (teona-pavliukova) [reply content/numpy/concepts/built-in-functions/terms/min/amin.md:35]: I couldn't fix this issue from a comment, so I deleted extra line in a separate commit ae99460.

**Comment by teona-pavliukova** [comment]
@caupolicandiaz Thank you for your review, all issues were fixed.

**Review by caupolicandiaz — APPROVED** [review]
@teona-pavliukova Looks good. I just made a couple of small edits to remove extra lines in the last code block.

File: content/numpy/concepts/built-in-functions/terms/min/amin.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`.amin()`** function returns the minimum of an array or minimum along an axis. The `a.min()` function is equivalent to `ndarray.min()`.
```
Comment (SSwiniarski) [inline content/numpy/concepts/built-in-functions/terms/min/amin.md:17]:
```suggestion
The **`.amin()`** function returns the minimum of an array or minimum along an axis. The `.amin()` function is equivalent to `ndarray.min()`.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@teona-pavliukova I did a review and had one comment.

**Review by SSwiniarski — APPROVED** [review]
@teona-pavliukova Looks good.

**Comment by SSwiniarski** [comment]
@teona-pavliukova Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/numpy/built-in-functions/amin

image

**Pull request merged**